IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcTranslationOperation/DocEntity.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcTranslationOperation" UniqueId="946a945b-8a4a-4e2c-9840-b56032727b5f" BaseDefinition="IfcCoordinateOperation" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="FirstCoordinate" UniqueId="f2534a2f-e119-4a52-b1cf-a55c170eea45" DefinedType="IfcMeasureValue">
+			<Documentation>The first coordinate of the translation. Can be a length measure in case of map coordinates and angle in case of geographic reference systems.</Documentation>
```

Comment:
```suggestion
			<Documentation>The first coordinate of the translation. Can be a length measure in case of map coordinates or a solid angle measure in case of geographic reference systems.</Documentation>
```